### PR TITLE
community/openjdk8: security upgrade to 3.9.0 (java 8u181b13)

### PR DESCRIPTION
**Pull request by J0WI**
Note: The location of version number has been changed in https://icedtea.classpath.org/hg/icedtea8/rev/03cdd2506cad

Please backport this change to older Alpine Linux releases.

### CODE REVIEW

**Comment by J0WI** [comment]
Fixes https://bugs.alpinelinux.org/issues/9314

**Comment by J0WI** [comment]
@fabled would be great if you could have a look here, because the Docker universe would welcome an update. :)

File: community/openjdk8/APKBUILD [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Timo Teras <timo.teras@iki.fi>
 pkgname=openjdk8
-_icedteaver=3.8.0
+_icedteaver=3.9.0
 # pkgver is <JDK version>.<JDK update>.<JDK build>
 # Check https://icedtea.classpath.org/wiki/Main_Page when updating!
-pkgver=8.171.11
+pkgver=8.181.13
 pkgrel=2
```
Comment (fabled) [inline community/openjdk8/APKBUILD]:
```suggestion
pkgrel=0
```

**Comment by J0WI** [comment]
@fabled thanks, I applied all requested changes.

**Comment by J0WI** [comment]
Please backport this change to older Alpine Linux releases.

**Comment by fabled** [comment]
Thanks. Applied squashed, and added fixes tag for the bug in the comment. Will look at which stable branches can be updated.

**Comment by fabled** [comment]
Backported to 3.8, 3.7 and 3.6.

**Comment by J0WI** [comment]
Thanks!
How about 3.5?

**Pull request closed without merging**